libmultilabel/linear/metrics.py

```diff
@@ -136,12 +175,50 @@ def get_metrics(metric_threshold: float,
                                  average=metric[:-3].lower(),
                                  multiclass=multiclass)
         else:
-            raise ValueError(f'Invalid metric: {metric}')
+            raise ValueError(f'invalid metric: {metric}')
 
     return MetricCollection(metrics)
 
 
+def compute_metrics(preds: np.ndarray,
```

Comment:
tutorials must be updated.. need to explain there the two settings (compute_metrics and update)